params/config.go

```diff
@@ -177,13 +177,21 @@ type WakuConfig struct {
 	// EnableRateLimiter set to true enables IP and peer ID rate limiting.
 	EnableRateLimiter bool
 
-	// RateLimitIP sets the limit on the number of messages per second
+	// PacketRateLimitIP sets the limit on the number of packets per second
 	// from a given IP.
-	RateLimitIP int64
+	PacketRateLimitIP int64
 
-	// RateLimitPeerID sets the limit on the number of messages per second
+	// PacketRateLimitPeerID sets the limit on the number of packets per second
 	// from a given peer ID.
-	RateLimitPeerID int64
+	PacketRateLimitPeerID int64
+
+	// BytesRateLimitIP sets the limit on the number of bytes per second
+	// from a given IP.
+	BytesRateLimitIP int64
+
+	// BytesRateLimitPeerID sets the limit on the number of bytes per second
+	// from a given peer ID.
+	BytesRateLimitPeerID int64
```

Comment:
Just a minor thing, but I really like this spacing style for consts, vars and fields. 👍